cubed/array_api/array_object.py

```diff
@@ -19,6 +19,7 @@
     float64,
 )
 from cubed.array_api.linear_algebra_functions import matmul
+from cubed.backend_array_api import namespace as nxp
```

Comment:
Why not simply `import namespace as np`? This would minimize the diff.

Reply:
It would, but I wanted to signal that this may not be regular NumPy. By using `nxp` here, it makes it easy to search for `np` to find places that are still using regular NumPy.